src/olympia/lib/settings_base.py

```diff
@@ -1345,14 +1345,24 @@ def get_db_config(environ_var, atomic_requests=True):
             'level': logging.WARNING,
             'propagate': False,
         },
+        'raven': {
+            'handlers': ['mozlog'],
+            'level': logging.WARNING,
+            'propagate': False
+        },
         'rdflib': {
             'handlers': ['null'],
-            'level': logging.DEBUG,
+            'level': logging.INFO,
             'propagate': False,
         },
+        'request': {
+            'handlers': ['mozlog'],
+            'level': logging.WARNING,
+            'propagate': False
+        },
         'request.summary': {
             'handlers': ['mozlog'],
-            'level': logging.DEBUG,
+            'level': logging.INFO,
             'propagate': False
```

Comment:
I suppose this is the default config now (inherited from `root`) so we could probably remove this entry.

Reply:
yes

Reply:
double check that `request.summary` doesn't inherit from `request`

Reply:
looks like this one comes from dockerflow so it shouldn't inherit from `request`